app/models/user.rb

```diff
@@ -0,0 +1,41 @@
+class User < ApplicationRecord
+  attr_accessor :skip_password_validation
+
+  rolify
+
+  extend FriendlyId 
+
+  friendly_id :friendly_field, use: :slugged
```

Comment:
it is not necessary to user slug with users because they have the email that must be uniq in the universe.

Reply:
I input this field, because in the page when de admin user show her projects is more friendly show this field in the params route instead to show the user-id

Reply:
ok, that's right, is not bad